hw/ip/rstmgr/data/rstmgr.hjson.tpl

```diff
@@ -107,6 +129,77 @@
       ]
     },
 
+    { name: "ALERT_INFO_CTRL",
+      desc: '''
+            Alert info dump controls.
+            ''',
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "EN",
+          hwaccess: "hrw",
+          desc: '''
+            Enable alert dump to capture new information.
+            This field is automatically set to 0 upon system reset (even if rstmgr is not reset).
+            '''
+          resval: "0"
+        },
+
+        { bits: "4+IdxWidth-1:4",
+          name: "INDEX",
+          desc: '''
+            Controls which 32-bit value to read.
+            '''
+          resval: "0"
+        },
+      ]
+    },
+
+    { name: "ALERT_INFO_ATTR",
+      desc: '''
+            Alert info dump attributes.
+            ''',
+      swaccess: "ro",
+      hwaccess: "hwo",
+      hwext: "true",
+      fields: [
+        { bits: "IdxWidth-1:0",
+          name: "CNT_AVAIL",
+          swaccess: "ro",
+          hwaccess: "hwo",
+          desc: '''
+            The number of 32-bit values contained in the alert info dump.
+            '''
+          resval: "0",
+          tags: [// This field only reflects the status of the design, thus the
+                 // default value is likely to change and not remain 0
+                 "excl:CsrAllTests:CsrExclCheck"]
```

Comment:
will the reset value be 0? If so maybe we can mart it as `excl:CsrNonInitTests:CsrExclCheck`.
The csr init test is csr_hw_reset, which will: write registers with rand value -> reset -> read registers and compare with expected reset value.

Reply:
Thanks for clarifying that @tjaychen I think this exclusion should be fine then:) No need to update that.